internal/indexer/definitions/tokyotoshokan.yaml

```diff
@@ -0,0 +1,63 @@
+---
+#id: tokyotosho
+name: Tokyo Toshokan
+identifier: tokyotosho
+description: A BitTorrent Library for Japanese Media
+language: en-us
+urls:
+  - https://
+privacy: public
+protocol: torrent
+supports:
+  - irc
+
+irc:
+  network: Rizon
+  server: irc.rizon.net
+  port: 6697
+  tls: true
+  channels:
+    - "#tokyotosho"
+  announcers:
+    - "TokyoTosho"
+  settings:
+    - name: nick
+      type: text
+      required: true
+      label: Nick
+      help: Bot nick. Eg. user_bot
+
+    - name: auth.account
+      type: text
+      required: false
+      label: NickServ Account
+      help: NickServ account. Make sure to group your main user and bot.
+
+    - name: auth.password
+      type: secret
+      required: false
+      label: NickServ Password
+      help: NickServ password
+  parse:
+    type: multi
+    lines:
+      - test:
+          - "Release 1748839: [Anime] SOMETITLE-007"
+        pattern: '^Release [0-9]*: \[(.*?)\] (.*)'
+        vars:
+          - category
+          - torrentName
+      - test:
+          - "Torrent: https://example.com/torrents/sha123/torrentname-007.torrent"
+        pattern: '^Torrent: (https:\/\/)(.*\.torrent)'
```

Comment:
I've been monitoring the channel for a couple of days and some urls does not have the `.torrent` so changing pattern to `^Torrent: (https:\/\/)(.*)` would be better.

And it could be simplified more to not capture the scheme: `^Torrent: https:\/\/(.*)` and then remove `- baseUrl`.

Reply:
Good point! I have updated the regex and removed `baseUrl` 👍